async_service/types.py

```diff
@@ -35,6 +35,14 @@ class InputFetchAckFailure(Exception):
     ...
 
 
+class MessageProcessFailure(Exception):
```

Comment:
Not required. We should not raise any exception if message processing had failed.